std/experimental/allocator/building_blocks/free_list.d

```diff
@@ -114,8 +114,7 @@ struct FreeList(ParentAllocator,
             _max = high;
         }
 
-        ///
-        @safe unittest
+        @system unittest
```

Comment:
Because it was never executed, see the comment about unittest blocks in templates before. By moving the unittest to the module level, this unittest was executed for the first time...

Reply:
Quite unfortunate. Thanks for the clarification.